docs/3 Scaffold a Blockchain/5 Message Scaffold.md

```diff
@@ -0,0 +1,28 @@
+# Message Scaffold
+
+Cosmos SDK messages are used to modify the state of a blockchain. Messages are bundled into transactions, broadcasted transactions are bundled into blocks and blocks make a blockchain.
+
+Starport supports scaffolding Cosmos SDK messages.
+
+```
+starport message [name] [field1] [field2] ... [flags]
+```
+
+Files and directories created and modified by scaffolding:
+
+* `proto`: message type
+* `x/module_name/keeper`: gRPC message server
+* `x/module_name/types`: message type, keys
```

Comment:
Users can see what Starport has scaffolded by running `git status`. I don't want to hard-code what we scaffold in the docs, because it might become out-of-date fast.